python/lsst/pipe/tasks/calibrateImage.py

```diff
@@ -1027,6 +1201,7 @@ def _apply_photometry(self, exposure, background):
 
         assert photo_calib._isConstant, \
             "Background calibration assumes a constant PhotoCalib; PhotoCalTask should always return that."
+        # FIXME: what to do about background_to_photometric_ratio here?
```

Comment:
I think "nothing", since we don't scale the persisted background model by the ratio when we do not apply the photometry to the image, either.  And that establishes the convention that if you want to add the background back in, you have to use the context manager.

The opposite convention - where the background subtraction task fits the background and then divides it by the `backgroundToPhotometricRatio` in order to let downstream code add it back in directly to a photometric-flattened image - might be less error-prone downstream, but I don't think the `BackgroundList` class is up to that, so let's be pragmatic and choose the convention we can actually implement right now.